### PR TITLE
chore: enable Swift CI tests

### PR DESCRIPTION
**Pull request by FUSAKLA**
Signed-off-by: Martin Chodur <m.chodur@seznam.cz>

Added Swift CI tests as suggested in https://github.com/thanos-io/thanos/issues/2829

I'm not sure if it should run for every test? I'm not sure if we are somehow limited in the CircleCI usage.
From time perspective it does not significantly prolong the test stage in CI.

### CODE REVIEW

**Comment by FUSAKLA** [comment]
@GiedriusS PTAL

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,13 +29,21 @@ jobs:
             fi
       - run:
           name: "Run unit tests."
+          environment:
+            THANOS_TEST_OBJSTORE_SKIP: AZURE,COS,ALIYUNOSS
+            # Variables for Sift testing.
```
Comment (bwplotka) [inline .circleci/config.yml:34]:
```suggestion
            # Variables for Swift testing.
```

**Comment by bwplotka** [comment]
Flaky test, rerun

**Comment by FUSAKLA** [comment]
Sorry for the lag I was on vacation. Thanks for fixing the typo and merging! :)

**Pull request merged**